website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -1693,6 +1693,8 @@ $ curl \
 
 This endpoint force unlocks a workspace. Only users with admin access are authorized to force unlock a workspace.
 
+For a force unlock, Terraform Cloud will attempt to find the latest provisional state version that has been finalized and set it as the current state version for a workspace. It is possible that an earlier provisional state version is selected as the current version for a workspace. This only occurs if the latest state version has not been finalized. Terraform Cloud will not prevent a force unlock if there are no available provisional state versions that have been finalized.
```

Comment:
Suggestion for brevity (please correct me if this isn't technical correct though!):
```suggestion
For a force unlock, Terraform Cloud attempts to find the latest finalized provisional state version and sets it as the current state version for a workspace. Terraform Cloud may select an earlier provisional state as the current version for a workspace. This only occurs if Terraform Cloud has not yet finalized the latest state version. Terraform Cloud will not prevent a force unlock if there are no available finalized provisional state versions.
```